src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -976,10 +976,24 @@ type FsharpLspServer(commands: Commands, lspClient: FSharpLspClient) =
             let source = String.concat "\n" lines
             let parsingOptions = Utils.projectOptionsToParseOptions opts
             let checker : FSharpChecker = commands.GetChecker()
+            let config =
```

Comment:
as it stands changes to this file are reread every time. Could you make a TODO or ENHANCEMENT comment here that we could watch this file for changes and store the changed value for reuse instead?

Reply:
Yep, left that part intentional unoptimized.
I've added the comment.